[Doc] Add more steps to getting started tutorial (#206)
liqotech/liqo: docs/pages/User/GettingStarted/_index.md

## The fast path to Liqo

The following steps will guide you through a tour to learn how to install Liqo and experiment with its most notable capabilities.

* [Install Liqo](./install): install Liqo on a first cluster (*home* cluster).
* [Peer to a foreign cluster](./peer): establish a peering with a second Liqo cluster (*foreign* cluster).
  * If you do not know who to peer with, you may need to install Liqo on a second cluster.
* [Exploit foreign resources](./test): start an *Hello World* application to verify that the two peered clusters can actually share resources correctly and that you are able to run a pod in a foreign cluster.

> * [Deploy a complex application](./further-steps): deploy a complex micro-services based application, to better show the Liqo capabilities.

Reviewer: ```suggestion
* [Deploy a complex application](./further-steps): deploy a complex application based on multiple micro-services, to demonstrate the advanced capabilities of Liqo.
```
